CRM/Export/Form/Select.php

```diff
@@ -120,15 +120,7 @@ public function preProcess() {
         break;
 
       default:
-        // FIXME: Code cleanup, we may not need to do this $componentName code here.
-        $formName = CRM_Utils_System::getClassName($this->controller->getStateMachine());
-        $componentName = explode('_', $formName);
-        if ($formName == 'CRM_Export_StateMachine_Standalone') {
-          $componentName = ['CRM', $this->controller->get('entity')];
-        }
-        // Contact
-        $entityShortname = $componentName[1];
-        $entityDAOName = $entityShortname;
+        $entityShortname = $entityDAOName = $this->getComponentName();
```

Comment:
This is incorrect; those two variables are not always the same, see for example `CRM_Export_Controller_Standalone::$components['Membership'] == 'Member'`

Granted the names of these variables is confusing. By "short name" they actually mean "component name"

Reply:
@colemanw can you push up a fix to address that? same pattern I think (FWIW I think the whole switch could be in those functions....

Reply:
@eileenmcnaughton stepping through this code, every condition in this switch statement appears to be unreachable except for the `default`. I put a breakpoint in each of the switch conditions and couldn't trigger any of them, using standalone export or the various component search screens.

Are you seeing the same thing? Can we just remove the whole `switch()` and always use the default condition?